src/code.h

```diff
@@ -1069,6 +1103,10 @@ void CodeListExprEnd(UInt nr, UInt range, UInt top, UInt tilde);
 */
 void CodeStringExpr(Obj str);
 
+UInt getNextFloatExprNumber(void);
```

Comment:
I am really unhappy about exposing this. I'll think about some alternatives / better APIs.